lib/manageiq/schema/engine.rb

```diff
@@ -3,6 +3,11 @@ module Schema
     class Engine < ::Rails::Engine
       isolate_namespace ManageIQ::Schema
 
+      ActiveSupport.on_load(:active_record) do
+        require 'migrate_with_cleared_schema_cache'
```

Comment:
Why require again?

Reply:
I'm thinking remove the require in schema.rb

Reply:
Done